api/v1alpha1/akodeploymentconfig_webhook.go

```diff
@@ -230,19 +247,33 @@ func (r *AKODeploymentConfig) validateAviSecret(secret *corev1.Secret, secretRef
 
 // validateAviControllerVersion checks NSX Advanced Load Balancer controller version valid or not
 func (r *AKODeploymentConfig) validateAviControllerVersion() (string, *field.Error) {
-	controllerVersion := AVI_VERSION
+	controllerVersion := ""
 	if r.Spec.ControllerVersion != "" {
```

Comment:
what if the user provides non-empty avi version, but it doesn't match what is returned in `AviSession.GetControllerVersion()`? show a warning?

Reply:
We will just put the right version in akodeploymentconfig, it is just like the mutating wehbook.